app/errors.js

```diff
@@ -0,0 +1,5 @@
+exports.handleNotFoundError = (err, req, res, next) => {
+  if (err.code === "42P01") {
+    res.status(404).send({ msg: err.msg });
+  } else next(err);
+};
```

Comment:
We should also have a 'handleServerError' error handler, which will act as a catch all for when we want to debug what our error is. This should log what the error is as well as returning a 500 error code and a 'server error' message.